<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CContactsView</class>
 <widget class="QWidget" name="CContactsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="4">
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>A&amp;dd</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lastName"/>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="address"/>
   </item>
   <item row="1" column="3">
    <widget class="QLineEdit" name="email"/>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&amp;Last Name</string>
     </property>
     <property name="buddy">
      <cstring>lastName</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Add&amp;ress</string>
     </property>
     <property name="buddy">
      <cstring>address</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QPushButton" name="changeButton">
     <property name="text">
      <string>&amp;Change</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&amp;E-Mail</string>
     </property>
     <property name="buddy">
      <cstring>email</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="4">
    <widget class="QTreeView" name="contacts"/>
   </item>
   <item row="2" column="4">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>First Name</string>
     </property>
     <property name="buddy">
      <cstring>firstName</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="firstName"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>firstName</tabstop>
  <tabstop>lastName</tabstop>
  <tabstop>address</tabstop>
  <tabstop>email</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>changeButton</tabstop>
  <tabstop>contacts</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
